data/sql/updates/pending_db_world/rev_1688549401806530816.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM `game_event_creature` WHERE `guid` IN (245648,245645,245649); 
+DELETE FROM `creature` WHERE `guid` IN (245648,245645,245649);
```

Comment:
Should the larger 'guid' be deleted?
```sql
DELETE FROM `creature` WHERE `guid` IN (248584, 248585, 248581);
DELETE FROM `game_event_creature` WHERE `guid` IN (248584, 248585, 248581);
```

Reply:
@pklloveyou I wasn't sure which one to delete. But I found that the NPCs with the larger GUIDs are located somewhere near the Bonfire, like actual Wardens. But the other GUIDs correspond to NPCs that are _inside_ the Bonfire, being burned (actually, just clipping through the asset). So, I decided to keep the ones that where located in a place that makes sense to me. 
But please, feel free to check if that's correct.